src/solar.js

```diff
@@ -518,130 +536,131 @@ var checkRaycasting = (mouse) => {
+//   document.getElementById("mySidenav").style.width = "0";
+// });
+//
+// document.getElementById("openPlanetsNav").addEventListener("click", function(){
+//   document.getElementById("planetsNav").style.width = "250px";
+// });
+//
+// document.getElementById("closePlanetsNav").addEventListener("click", function(){
+//   document.getElementById("planetsNav").style.width = "0px";
+// });
+//
+// document.getElementById("showSun").addEventListener("click", function(){
+//   sunSelect();
+// });
+//
+// document.getElementById("showMoon").addEventListener("click", function(){
+//   moonSelect();
+// });
+//
+// document.getElementById("showMercury").addEventListener("click", function(){
+//   planetSelect(0);
+// });
+//
+// document.getElementById("showVenus").addEventListener("click", function(){
+//   planetSelect(1);
+// });
+//
+// document.getElementById("showEarth").addEventListener("click", function(){
+//   planetSelect(2);
+// });
+//
+// document.getElementById("showMars").addEventListener("click", function(){
+//   planetSelect(3);
+// });
+//
+// document.getElementById("showJupiter").addEventListener("click", function(){
+//   planetSelect(4);
+// });
+//
+// document.getElementById("showSaturn").addEventListener("click", function(){
+//   planetSelect(5);
+// });
+//
+// document.getElementById("showUranus").addEventListener("click", function(){
+//   planetSelect(6);
+// });
+//
+// document.getElementById("showNeptune").addEventListener("click", function(){
+//   planetSelect(7);
+// });
+//
+// document.getElementById("showPluto").addEventListener("click", function(){
+//   planetSelect(8);
+// });
+//
+// document.getElementById("toggleAstronaut").addEventListener("click", function(){
+//   jsonObj.astronaut.visible=!(jsonObj.astronaut.visible);
+//   console.log("astronaut.visible = ", jsonObj.astronaut.visible);
+// });
+
+// document.getElementById("toggleTrace").addEventListener("click", function(){
+//   //jsonObj.showPlanetLines=!(jsonObj.showPlanetLines);
+//   //console.log("showPlanetLines = ", jsonObj.showPlanetLines);
+//   if(jsonObj.showPlanetLines){
+//     jsonObj.showPlanetLines = false;
+//     for (let i=0; i<jsonObj.numPlanets; i++){
+//       scene.remove(orbitLines[i]);
+//     }
+//   }
+//   else if(!jsonObj.showPlanetLines){
+//     jsonObj.showPlanetLines = true;
+//     for (let i=0;i<jsonObj.numPlanets; i++){
+//       scene.add(orbitLines[i]);
+//     }
+//   }
+// });
 
 // Start of Range slider collapsible content
-var coll = document.getElementsByClassName("collapsible");
-var i;
-
-for (i = 0; i < coll.length; i++) {
-  coll[i].addEventListener("click", function() {
-    this.classList.toggle("active");
-    var content = this.nextElementSibling;
-    if (content.style.display === "block") {
-      content.style.display = "none";
-    } else {
-      content.style.display = "block";
-    }
-  });
-}
+// let coll = document.getElementsByClassName("collapsible");
```

Comment:
It's usually not a very good practice to land commented code (especially these big chunks). One can always look at the git history and copy paste back.